### PR TITLE
Set crypto library dependency to always use the lastest available

### PR DESCRIPTION
**Pull request by inetic**

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @benjaminbollen

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/sentinel.rs [hunk 1]
```diff
@@ -209,7 +209,7 @@ impl<'a, Request, Name>
                 // We use raw bytes from the key here because in the current
                 // version of sodiumdioxide library PublicKey doesn't derive
                 // from Eq nor PartialEq. Once a version greater or equal to
-                // 0.0.5 is used, we can get rid of this unwrapping and then
+                // 0.0.6 is used, we can get rid of this unwrapping and then
```
Comment (benjaminbollen) [inline src/sentinel.rs:212]: Thanks, can you make a ToDo list time for post-sprint to follow up on this (maybe not yet a JIRA task)?

Comment (inetic) [inline src/sentinel.rs:212]: Where would that be? In the same README.md file?

Comment (benjaminbollen) [inline src/sentinel.rs:212]: yes, but maybe don't add it as a sprint task, because we don't control when sodiumoxide will bump the version

Comment (inetic) [inline src/sentinel.rs:212]: So what name of the subsection of the Todo section would you suggest? (Also should be OK'd by @Viv-Rajkumar I guess to keep some standard).

A second option would be if we just mark the comment as FIXME and whoever stumbles at the code can check it whether it's been added into the library?

And a third option would be to add the TODO into the Cargo.toml right where version of sodiumdioxide is specified. Just found out it should be fixed to 0.0.5 as the new version probably won't have asymmetrixbox.

I don't mind either way.

Comment (benjaminbollen) [inline src/sentinel.rs:212]: Yeah, Im not too fussed about it.  We'll pick up on it.  So maybe just drop it and move on to the next task.

**Comment by benjaminbollen** [comment]
Just revert it to 0.0.5 explicitly then and add a FixMe, sorry for merging too fast :)

**Pull request merged**